torch/_inductor/decomposition.py

```diff
@@ -79,6 +79,7 @@
 decomps_to_exclude = [
     aten._unsafe_index,
     aten._scaled_dot_product_flash_attention_for_cpu.default,  # See comments in torch/_decomp/decompositions.py
+    aten._softmax_backward_data,
```

Comment:
why are you excluding it?

Reply:
If I don't exclude the core decomposition it would report a decomposition is already registered when it comes to register the inductor-specific decomp with the fma operation.

Reply:
ah, I see, we already have a decomp for this one.